app/controllers/admin/tags_controller.rb

```diff
@@ -6,6 +6,7 @@ class TagsController < Admin::ApplicationController
       id supported rules_markdown short_summary pretty_name bg_color_hex
       text_color_hex user_id alias_for badge_id requires_approval
       category social_preview_template wiki_body_markdown submission_template
+      name
```

Comment:
By not including `name` within the `ALLOWED_PARAMS` constant, Admins were unable to `create` and `update` tags via `/admin/content_manager/tags`. The logs would silently spit out an `Unpermitted param :name` error, but there was no indication that the creation or updating of the tag had failed--it just appeared that the "create" and "update" buttons were disabled.